core/src/utils/animation/animation-utils.ts

```diff
@@ -24,12 +24,25 @@ const convertHyphenToCamelCase = (str: string) => {
   return str.replace(/-([a-z])/ig, g => g[1].toUpperCase());
 };
 
+let animationPrefix: string | null = null;
+
+export const getAnimationPrefix = (el: HTMLElement): string => {
+  if (animationPrefix === null) {
```

Comment:
See comment above.
```suggestion
  if (animationPrefix === undefined) {
```

Reply:
I do not know what the suggestion will do or prevent? How the animationPrefix can be undefined?  It is null when not initial assigned otherwise an empty string or '-webkit-'